articles/machines/qc-trace-simulator/distinct-inputs-checker.md

```diff
@@ -12,9 +12,8 @@ uid: microsoft.quantum.machines.qc-trace-simulator.distinct-inputs
 # Distinct Inputs Checker
 
 The `Distinct Inputs Checker` is a part of the quantum computer [Trace
-Simulator](xref:microsoft.quantum.machines.qc-trace-simulator.intro). It is designed for detecting
-potential bugs in the code. Consider the following piece of Q# code to
-illustrate the issues detected by this package:
+Simulator]<xref:microsoft.quantum.machines.qc-trace-simulator.intro>. It is designed to detect
```

Comment:
Markdown links should always appear as `[link text](link target)`. The angle bracket notation is used only when the link text should  be implied from the link target.
```suggestion
Simulator](xref:microsoft.quantum.machines.qc-trace-simulator.intro). It is designed to detect
```